.eslintrc.js

```diff
@@ -55,6 +55,7 @@ module.exports = {
     'default-param-last': 'off',
     'prefer-object-spread': 'error',
     'require-atomic-updates': 'off',
+    'curly': 'error',
```

Comment:
This is already set in `@metamask/eslint-config`, to `['error', 'all']`.  Can we use that instead? It's not clear to me why it's not already enabled :thinking: I don't see it being disabled.